libs/deer/.config/lints.toml

```diff
@@ -0,0 +1,14 @@
+# deer specific lints
```

Comment:
Are you going to remove these lints from `lib.rs` then?

Reply:
removed in [e729f4d](https://github.com/hashintel/hash/pull/2075/commits/e729f4dbc50b9b88d9dad7091bad61eacb5e8783)